x-pack/plugins/cloud_security_posture/server/lib/telemetry/collectors/installation_stats_collector.ts

```diff
@@ -34,11 +34,11 @@ export const getInstallationStats = async (
     isPluginInitialized,
   };
 
-  const getInstalledPackagePolicies = async (
+  const getInstalledPackagePolicies = (
```

Comment:
it appears there are async calls as part of `getInstalledPackagePolicies` implementation
It should be `async`

Reply:
my bad